bench-utils/src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub mod inner {
                 end_info,
                 message,
                 final_time,
-                pad = 75 - indent_amount
+                pad = 75 - indent_amount.min(75)
```

Comment:
Would be best to fix all the timers to make sure we close them in error case. Let's open an issue maybe